CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
+- Added `safe_onnx_export` function with workarounds for `onnx_ir.serde.SerdeError` issues in ONNX export ([#XXXX](https://github.com/pyg-team/pytorch_geometric/pull/XXXX))
```

Comment:
@AJamal27891 @puririshi98 The PR number is not filled here. Could either of you send another PR?